code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -110,8 +110,7 @@ Difficulty: Very Hard
 /obj/effect/temp_visual/at_shield/New(new_loc, new_target)
 	..()
 	target = new_target
-	spawn(0)
-		orbit(target, 0, FALSE, 0, 0, FALSE, TRUE)
+	INVOKE_ASYNC(src, /atom/movable/proc/orbit, target, 0, FALSE, 0, 0, FALSE, TRUE)
```

Comment:
are you sure this will work? Usually you need a ./proc

Reply:
this is how TG has it, i'll test it again to ensure it works

Reply:
it seems to work fine, the shield was orbiting the colossus as I shot at him